groups/staging-windows-virtual-machines/locals.tf

```diff
@@ -49,6 +49,11 @@ locals {
 
   bus_obj_2_log_groups = compact([for log, map in local.bus_obj_2_cw_logs : lookup(map, "log_group_name", "")])
 
+  bus_obj_2_volume_snapshot_ids = jsondecode(local.bus_obj_2_ec2_data["volume_snapshot_ids"])
+  bus_obj_2_xvdf_snapshot_id    = try(local.bus_obj_2_volume_snapshot_ids["xvdf"], null)
```

Comment:
It might be better to use lookup with a default rather than a try/fail? i.e. `lookup(local.bus_obj_2_volume_snapshot_ids, "xvdf", null)`